angular-konami.js

```diff
@@ -18,7 +18,7 @@ angular.module('konami', [])
 
         // Let user define a custom code.
         if (attr.konamiKeys) {
-          var konamiKeys = scope.$eval(attr.konamiKeys);
```

Comment:
The use of "var" here is not optional; that's how we allow overriding konamiKeys for a directive instance, but keep the default available on the outer scope for future directive instances.